client/integrations/integrations.go

```diff
@@ -432,29 +432,41 @@ func List(pageSize int, pageToken string, filter string, orderBy string) (respBo
 }
 
 // Get
-func Get(name string, version string, basicInfo bool) ([]byte, error) {
+func Get(name string, version string, basicInfo bool, orides bool) ([]byte, error) {
```

Comment:
can. you please name the variable expanded? "overrides"

Reply:
I will just use "override" as use of  "overrides" conflict with type "overrides" used later in same code block.